RecoJets/JetProducers/interface/FastjetJetProducer.h

```diff
@@ -0,0 +1,118 @@
+#ifndef FastjetJetProducer_h
+#define FastjetJetProducer_h
```

Comment:
Please use an include guard matching the full file name subsystem_package_fileName_h